Quotient/e2ee/qolmaccount.h

```diff
@@ -67,7 +67,7 @@ class QUOTIENT_API QOlmAccount : public QObject
     //! Sign all one time keys.
     OneTimeKeys signOneTimeKeys(const UnsignedOneTimeKeys &keys) const;
 
-    UploadKeysJob* createUploadKeyRequest(const UnsignedOneTimeKeys& oneTimeKeys) const;
+    UploadKeysJob *createUploadKeyRequest(const UnsignedOneTimeKeys& oneTimeKeys) const;
```

Comment:
that doesn't look right

Reply:
What do you mean exactly? The function creates a job object but leaves it to the caller to actually start. I'm not sure it belongs to `QOlmAccount`, it may make sense to move it out to `Connection` instead. It's mainly used from `TestOlmAccount`, we could make it a friend, in order to not expose the function externally. And then it can become a proper job invocation instead of just preparation. Is that what you mean, or something different?

Reply:
Oh, wait, the code formatting does not look right indeed.